server/events/tests/test_models.py

```diff
@@ -0,0 +1,36 @@
+from datetime import timedelta
+from django.utils import timezone
+from django.contrib.auth import models as auth_models
+from .shared_case import SharedTestCase
+from events.models import Event
+
+class EventTest(SharedTestCase):
+    """ Test module for Event model """
+
+    def setUp(self):
+        # event1
```

Comment:
I've always been of the opinion that comments should be used for explaining _why_ you're doing something, rather than _what_ you're doing. If you have to explain _what_ you're doing in a comment, try to use more descriptive variable names or breaking code up into functions.

In this case, try just setting these calls to variable names. It'll help testcases later!

Reply:
Fixed